docs/device-tree-overlay/intro.md

```diff
@@ -1,68 +1,70 @@
 ---
-title: Device Tree Overlays Loadable at Runtime
+title: Device Tree Overview
 ---
 
 import DocCardList from '@theme/DocCardList';
 
-<!-- -> Why is this important
-    - see https://github.com/OnionIoT/OpenWRT-Packages/issues/62 -->
+# Device Tree Overview
 
-The OpenWRT-22.03 based firmware supports runtime loading of Device Tree Overlays. 
+## Introduction
+The OpenWRT-22.03 based firmware supports runtime loading of Device Tree Overlays.
 
-This means users are able to dynamically add hardware support to a running firmware just by installing packages, opening the door for easier support of more use cases for the Omega2.
+This means you can dynamically add hardware support to a running firmware just by installing packages, opening the door for easier support of more use cases for the Omega2.
 
-With this, users can avoid:
+With this, you can avoid:
 
-* Having to build fully custom firmware to configure specific hardware functionality
-* Having to maintain multiple DTS files
-* Having to use the Build System to create firmware for scratch - which can take hours
+- Having to build fully custom firmware to configure specific hardware functionality.
+- Having to maintain multiple DTS files.
+- Having to use the Build System to create firmware for scratch - which can take hours.
 
-Instead they can:
+Instead, you can:
+- Change device functionality just by installing a package and rebooting.
+- Quickly build firmware that supports customized hardware using the Image Builder.
 
-* Change device functionality just by installing a package and rebooting
-* Quickly build firmware that supports customized hardware using the Image Builder 
+## Context: What are Device Trees and how do they work?
+The Device Tree is a data structure that describes the hardware components of a computer or device for the operating system's kernel to use and manage those components. This includes the CPU, memory, storage, and all peripherals.
 
-<details>
-  <summary>
-
-**Context: What are Device Trees? How do they generally work?**
-  </summary>
-  <div>
-    
-The Device Tree is a data structure that describes the hardware components of a computer or device in order for the operating system's kernel to use and manage those components. This includes the CPU, memory, storage, and all peripherals. 
-
-For more info, check out the Device Tree for Dummies presentation by Thomas Petazzoni: https://elinux.org/images/f/f9/Petazzoni-device-tree-dummies_0.pdf
+For more info, check out the [Device Tree for Dummies](https://elinux.org/images/f/f9/Petazzoni-device-tree-dummies_0.pdf) presentation by Thomas Petazzoni.
 
 ### Device Trees in OpenWRT
+By default, OpenWRT supports static Device Trees in the form of Device Tree Specification (DTS) files. For example, all the DTS files for ramips-based devices can be found on [GitHub's OpenWRT](https://github.com/openwrt/openwrt/tree/main/target/linux/ramips/dts) repository.
```

Comment:
For the link, it would make more sense to say "OpenWRT's build system GitHub repo" instead of "GitHub's OpenWRT"